codeforamerica/vita-min
# add return_to_review hidden field to az public school contribution form

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1426
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Added a hidden field to the az public school contribution form that makes sure the "return_to_review" param is kept after a contribution is edited. This was already in place on the qualifying org contribution and unemployment forms, but was missing here.
## How to test?
- Go through the AZ flow, adding a public school contribution. From the final review page, click edit on public school contributions. On the public school contributions list, hit edit on the contribution you created earlier. Edit some field and submit the form to get back to the public school contributions list. Hit Continue and verify that you go directly back to the final review screen.
- Go on demo and verify that by following the same instructions, you instead proceed to the screen after public school contributions in the flow.
